tools/replaceColumn/replaceColumn.xml

```diff
@@ -0,0 +1,99 @@
+<tool id="replace_column_script" name="Replace columns" version="0.1">
+    <description>Replace the values of a column by values which are defined in a convert file.</description>
+    <requirements>
+        <requirement type="package" version="2.7">python</requirement>
+    </requirements>
+    <command>
+        <![CDATA[
+        python "$replaceColumnScript" '$inputs'
+        ]]>
+    </command>
+    <configfiles>
+        <inputs name="inputs" />
+        <configfile name="replaceColumnScript">
+            <![CDATA[
+import sys
+
+replace_file = "$replace_information"
+original_file = "$original_file"
+column = int("$column_replace") - 1 ## assume that user thinks column 0 is column 1
+ignore_start_lines = int("$ignore_lines")
+
+if column < 0:
+    print('No negative column id possible! Columns IDs have the minimal value of \'1\'')
+    sys.exit(1)
+
+## read conversion information to index 
+conversion = {}
+
+with open(replace_file, 'r') as conversion_file:
+    for line in conversion_file:
+        conv_key_value = line.split()
+        if len(conv_key_value) == 2:
+            conversion[conv_key_value[0].strip('\n')] = conv_key_value[1].strip('\n')
+        else:
+            conversion[conv_key_value[0].strip('\n')] = ''
+                    
+
+## read file line by line, search for column entry if it can be replaced. Otherwise it will be skipped.
+with open("output_file", 'w') as output:
+    count = 0
+
+    with open(original_file) as original:
+
+        for line in original:
+            write = True
+            if count < ignore_start_lines:
```

Comment:
I keep the lines but let them unmodified. I think it is user-friendly to say 'Ignore first n lines here' in this tool instead of choosing another tool just to remove these lines.